tools/docs/github-query-items.ts

```diff
@@ -46,6 +29,34 @@ export interface Items {
   features: ItemsEntity[];
 }
 
+const GhOutputSchema = z.array(
+  z.object({
+    url: z.string(),
+    title: z.string(),
+    labels: z.array(
+      z.object({
+        name: z.string(),
+      }),
+    ),
+    number: z.number(),
+  }),
+);
+
+async function getIssuesByIssueType(
+  issueType: 'Bug' | 'Feature',
+): Promise<ItemsEntity[]> {
+  const command = `gh issue list --json "title,number,url,labels" --search "type:${issueType}" --limit 1000`;
+  const execRes = await exec(command);
+  const res = GhOutputSchema.safeParse(JSON.parse(execRes.stdout));
+  if (res.error) {
+    throw res.error;
+  }
+
+  return res.data.map((issue) => {
+    return { ...issue, issueType };
```

Comment:
sad that it's not a property on result type 😔

Reply:
Yes, they are still working on the APIs